project_milestone_status/tests/test_project_milestone_status.py

```
@@ -0,0 +1,51 @@
+from odoo.addons.project_milestone_status.tests.common import (
+    ProjectMilestoneStatusCommon,
+)
+
+
+class TestProjectMilestoneStatus(ProjectMilestoneStatusCommon):
+    def test_check_execution_empty(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_check_execution_empty`

Add return type annotation: `None`

(ANN201)